src/status_im2/common/standard_authentication/standard_auth/button/view.cljs

```diff
@@ -34,7 +35,6 @@
                                                       :auth-button-icon-left auth-button-icon-left
```

Comment:
@J-Son89, probably something you realized at this point after having worked with the existing auth code. One of the reasons the code is convoluted is because I think we aren't using re-frame in the right spirit, i.e. we're breaking its architectural principles.

For instance, we call `authorize/authorize` in the UI layer, a function that calls functions with promises, hiding them behind callbacks, and which do further re-dispatches. Clearly, `authorize/authorize` should be decomposed as a pure event, and an effect that does the RN biometrics using the touch-id lib. The usual re-frame stuff.

I'm not suggesting changes in this PR because you are inheriting the debt and playing along, just something to keep in mind if you or somebody else touch auth code in a future PR.